utils/test/TestKickStart.bat

```diff
@@ -0,0 +1,4 @@
+
+@ECHO ON
+python TestRunner.py
```

Comment:
I think @JudTown17 had some issues getting it to run. That being said it should probably be added back in and just commented out. Judson is going to work on getting the tests to run from the top down during this sprint.

Judson: Add the previous comments back into the **.BAT** file and we'll work on what's happening on your end and add it to our testing instructions.